scripts/blockchain

```diff
@@ -13,22 +13,35 @@ def main():
     blockchain_path = pathlib.Path(__file__).absolute().parent.parent.joinpath("blockchain")
     blockchain_node_path = blockchain_path.joinpath("node_modules")
     blockchain_json_src_path = blockchain_node_path.joinpath("singularitynet-alpha-blockchain")
-    token_json_src_path = blockchain_node_path.joinpath("singularitynet-token-contracts", "SingularityNetToken.json")
+    token_json_src_dir = blockchain_node_path.joinpath("singularitynet-token-contracts")
+    token_json_file_name = "SingularityNetToken"
     blockchain_json_dest_path = pathlib.Path(__file__).absolute().parent.parent.joinpath("snet_cli", "resources", "contracts")
-    contracts = ["Agent", "AgentFactory", "Job", "Registry"]
+    abis = ["Agent", "AgentFactory", "Job", "Registry", "AlphaRegistry"]
+    networks = ["AgentFactory", "Registry", "AlphaRegistry"]
 
     if target == "install":
-        subprocess.call([shutil.which('npm'), "install"], cwd=blockchain_path)
-        for contract in contracts:
-            shutil.copy(blockchain_json_src_path.joinpath("{}.json".format(contract)),
-                        blockchain_json_dest_path.joinpath("{}.json".format(contract)))
-        shutil.copy(token_json_src_path, blockchain_json_dest_path.joinpath("SingularityNetToken.json"))
+        if not os.path.isdir(blockchain_json_src_path) and not os.path.isdir(token_json_src_dir):
```

Comment:
should be an `... or not ...` IMO -- we need to do an npm install if either package are missing